test/models/role_test.rb

```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
+class RoleTest < ActiveSupport::TestCase
+  # test "the truth" do
```

Comment:
We're not using this test file anywhere, but we should still commit it.
Please remove this file